instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -183,15 +212,28 @@ def get_or_create_headers():
                 result = getattr(exc, "response", None)
             finally:
                 elapsed_time = max(
-                    round((default_timer() - start_time) * 1000), 0
+                    round((default_timer() - start_time)), 0
                 )
                 context.detach(token)
 
             if isinstance(result, Response):
                 if span.is_recording():
-                    span.set_attribute(
-                        SpanAttributes.HTTP_STATUS_CODE, result.status_code
+                    additional_span_attributes = {
+                        SpanAttributes.HTTP_STATUS_CODE: result.status_code,
+                    }
+                    additional_span_attributes = _get_attributes_based_on_stability_mode(
+                        sem_conv_opt_in_mode,
+                        additional_span_attributes,
                     )
+                    version = ""
+                    if result.raw is not None:
+                        version = getattr(result.raw, "version", None)
+                        if version:
+                            version = "1.1" if version == 11 else "1.0"
```

Comment:
nit: no HTTP/2 or /3 support?

Reply:
NETWORK_PROTOCOL_VERSION is already a part of the mapping and so, should be set as part of _get_attributes_based_on_stability_mode. My guess is that this is covering some edge case. Will explore and if needed talk to Leighton about what the intent was when he returns.

Reply:
Requests library only supports HTTP/1 https://stackoverflow.com/questions/34570991/http-2-request-in-python-2-7#:~:text=As%20others%20mentioned%20in%20the%20comments%20to%20the,of%20is%20hyper%2C%20which%20quoting%20from%20the%20docs%3A